### ISSUE TITLE
Doc comments on namespaced types generate on the namespace and the type

### ISSUE THREAD

**Issue by brennie** [dbeckwith/rust-typescript-type-def#20]
Given a struct like:

```
#[derive(serde::Serialize, typescript_type_def::TypeDef)]
#[type_def(namespace = "foo")]
/// Description of Bar
struct Bar {
    pub field: String,
}
```

The following is generated:

```
/**
 * Description of Bar
 */
export namespace foo{export type Bar=
/**
 * Description of Bar
 */
{"field":string;};}
```

I would expect the "Description of Bar" comment to only be attached to the Bar type, not the namespace.

**Comment by brennie** [comment]
Even without the namespace, the generated type contains two doc comments:

```
/// Description of Bar
struct Bar {
    pub field: String,
}
```

generates

```
/**
 * Description of Bar
 */
export type Bar=
/**
 * Description of Bar
 */
{"field":string;};
```

which I find unexpected.

**Comment by dbeckwith** [comment]
Thanks for the report! Fixed in v0.5.7.

**Comment by brennie** [comment]
Thanks for the quick turnaround!
